### PR TITLE
Flag is --script not --scenario

### LINKED ISSUE #1561: node jobs are failing

### PR DESCRIPTION
**Pull request by fejta**
Closes #1561

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -232,14 +232,14 @@
 "pull-kubernetes-node-e2e": {
   "scenario": "kubernetes_kubelet",
   "args": [
-    "--scenario=test/e2e_node/jenkins/jenkins-pull.properties"
+    "--script=test/e2e_node/jenkins/jenkins-pull.properties"
```
Comment (liggitt) [inline jobs/config.json]: is it --script or --properties?

Comment (fejta) [inline jobs/config.json]: Definitely properties

**Comment by krzyzacy** [comment]
--properties... whoops...

File: jobs/config.json (hunk 1 above)
Comment (fejta) [inline jobs/config.json]: Apparently I lost the race between Ryan committing the change for me and my pushing the fix to this branch

**Comment by fejta** [comment]
Maybe I should go back to attending meetings!

**Pull request merged**